series-tmdb.en.yaml

```diff
@@ -2970,39 +3300,68 @@ entries:
         anilist-id: 11319
 
   - title: "The Faraway Paladin"
-    # https://www.themoviedb.org/tv/123525
-    # https://www.thetvdb.com/dereferrer/series/401915
-    # https://www.imdb.com/title/tt14476244/
+    # imdb: https://www.imdb.com/title/tt14476244/
+    # tmdb: https://www.themoviedb.org/tv/123525
+    # tvdb: https://www.thetvdb.com/dereferrer/series/401915
     synonyms:
       - "Paladin of the End"
       - "Saihate no Paladin"
       - "世界尽头的圣骑士"
       - "最果てのパラディン"
+    # guid: plex://show/607c5f8e7ce89d0035158fbf
     seasons:
       - season: 1
         anilist-id: 132473
       - season: 2
         anilist-id: 143085
 
+  - title: "The Garden of Sinners"
```

Comment:
I don't think we need this entry as most people probably have them as 1 episode per movie, which matches the movies-tmdb.en.yaml entry.

Reply:
Fine by me, it felt needlessly complicated but didn't wanna remove the existing entry on it, I'll remove it